docs/book/v3/cookbook/debug-toolbars.md

```diff
@@ -44,12 +44,78 @@ following contents:
 <?php
 
 use PhpMiddleware\PhpDebugBar\ConfigProvider;
+use Psr\Container\ContainerInterface;
 
-$provider = new ConfigProvider();
-return $provider();
+return array_merge(ConfigProvider::getConfig(), [
+    'dependencies' => [
+        'factories' => [
+            \PhpMiddleware\PhpDebugBar\PhpDebugBarMiddleware::class => \PhpMiddleware\PhpDebugBar\PhpDebugBarMiddlewareFactory::class,
+            \DebugBar\DataCollector\ConfigCollector::class => \PhpMiddleware\PhpDebugBar\ConfigCollectorFactory::class,
+            \PhpMiddleware\PhpDebugBar\ConfigProvider::class => function(ContainerInterface $container) {
+                return $container->get('config');
+            },
+            \DebugBar\DebugBar::class => \PhpMiddleware\PhpDebugBar\StandardDebugBarFactory::class,
+            \DebugBar\JavascriptRenderer::class => \PhpMiddleware\PhpDebugBar\JavascriptRendererFactory::class,
```

Comment:
To shorten these lines I suggest importing each of the classes, or the namespaces `PhpMiddleware\PhpDebugBar` and `DebugBar`, at the top of the file.